src/messaging/models.py

```diff
@@ -5,19 +5,26 @@
 # Create your models here.
 
 class Thread(models.Model):
-	profile_1 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_1')
-	profile_2 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_2')
```

Comment:
Were all these tabs wrong?

Reply:
spaces not tabs for Python. I personally like tabs too, but spaces is the default of the entire community